docs/iam/access-management-plugin.md

```diff
@@ -22,7 +22,7 @@ In another terminal start MinIO:
 export MINIO_CI_CD=1
 export MINIO_ROOT_USER=minio
 export MINIO_ROOT_PASSWORD=minio123
-export MINIO_POLICY_OPA_URL=http://localhost:8080/
```

Comment:
This value will still be supported @itsericqiu however using new ENV is correct.

Reply:
Got it. Thanks for for the clarification 👍